factory-reset/eos-factory-reset

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python3
+import argparse
+import sys
+import subprocess
+
+# Stages list
+# Each stage has the form (short option, name, description)
+# In addition from the stages listed bellow, the following stage names and
+# short option names are already taken by other command line arguments:
+# ('q', 'quiet', 'Suppress all output and don\'t ask for confirmation')
+# ('D', 'disable', 'Disable the selected stages instead of enabling them')
```

Comment:
I'd remove the option definitions from the comment, I guarantee you that once anyone adds a new option or change an existing one it's gonna forgot to update the comment :)

Reply:
Good point. I'm going to change this to two lists, one with stages and another with the non-stage arguments, than build the parser arguments from both lists. This way we have a single source for this info and a good visibility, to ease maintenance.